server/sonar-web/src/main/js/components/issue/components/IssueAssign.js

```diff
@@ -52,13 +52,13 @@ export default class IssueAssign extends React.PureComponent {
             <Avatar
               className="little-spacer-right"
               hash={issue.assigneeAvatar}
-              name={issue.assigneeName}
+              name={issue.assigneeName || issue.assignee}
               size={16}
             />
           </span>
         )}
         <span className="issue-meta-label">
-          {issue.assignee ? issue.assigneeName : translate('unassigned')}
+          {issue.assignee ? issue.assigneeName || issue.assignee : translate('unassigned')}
```

Comment:
I understand it does the job but it looks weird to me... Maybe doing `const name = issue.assigneeName || issue.assignee;` first would make it more subtle. That would also avoid a code repetition. No big deal tho.

Reply:
I won't fix this one, because of that type undefined that I told you about.